lookout/style/typos/corrector_manager.py

```diff
@@ -25,7 +25,8 @@ def get(self, source: str) -> TyposCorrector:
         """
         obj = self.correctors.get(source)
         if obj is None:
-            obj = TyposCorrector(threads_number=1).load(source)
+            obj = TyposCorrector().load(source)
+            obj.threads_number = 1
```

Comment:
I guess it should be `processes_number` now

Reply:
It will be changed in another PR, as this one is gonna be closed most likely (as part of it is contained in another PR and so on).